CHANGELOG.md

```diff
@@ -2,6 +2,120 @@
 
 ## Unreleased
 
+### Added
+Publish ods images to Docker Hub ([#490](https://github.com/opendevstack/ods-core/issues/490))
+Add script to manage Bitbucket ODS project and repos ([#614](https://github.com/opendevstack/ods-core/pull/614))
+Add make to the slave-base ([#539](https://github.com/opendevstack/ods-core/issues/539))
```

Comment:
Can you remove this? I took it off from ODS 3.0 as nothing was done here - `make` was always present